documentation/book/proc-configuring-kafka-mirror-maker-abort-on-send-failure.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-mirror-maker-abort-on-send-failure.adoc
+
+[id='proc-configuring-kafka-mirror-maker-abort-on-send-failure-{context}']
+= Enabling Mirror Maker to continue on message failure
+
+By default, if an error occurs when sending a message from Kafka Mirror Maker to a Kafka cluster:
+
+* The Kafka Mirror Maker container is terminated in {ProductPlatformName}.
+* The container is then recreated.
+
+To ignore the sending error and attempt to send the next message in the queue, you can set the `KafkaMirrorMaker.spec.producer.abortOnSendFailure` property to `false`.
```

Comment:
I would not use "queue" here, it's something that can be misleading in Kafka :-)
Let's just say "in the topic" or just "to send the next message,"

Reply:
I think that this also suggest that a message would be lost when this happens (since it says that it will attempt to send the _next message_). Is that really the case?

Reply:
Yes, that's the case. From Kafka MM source code, there is a matric tracing the number of dropped messages which is incremented every time the above happens.

Reply:
WARNING: Messages will be lost when setting this value to 'false`, as there is no attempt to resend a failed message.